### ISSUE TITLE
[FEATURE] Session Multi-Selection

### ISSUE THREAD

**Issue by radj307** [radj307/volume-control#88]
This feature was mentioned in #54 & #87.

Currently, if you want to adjust the volume of multiple sessions at the same time, you have to manually add target overrides for each session to a hotkey action. While this approach *does* work, it isn't very intuitive and is frustrating to use.  
One of these frustrations (beyond being annoying to configure) is that you have to use a different key combination for actions with target override(s) if you want to keep volume control's default hotkeys alongside them, since multiple hotkeys can't use the same key combination.

## Current Behaviour

Currently, 0 or 1 session(s) may be selected at any given time.  
The Next/Previous Session actions directly increment/decrement the selected session, and all other session actions *(without any configured target overrides)* affect the currently selected session **only**.
![image](https://github.com/radj307/volume-control/assets/1927798/42494b75-a59b-41f1-a827-bc3a4b4fd08a)  
Actions with at least one target override configured ignore the selected session entirely, and **only** affect the specified sessions.

## Proposed Behaviour

First of all, the concept of a *selected session* needs to be split up in order to make this easier to understand. Previously the term *selected session* was used to refer to the *selector* (which points to an item in the list) and the *target* (which is the actual item pointed to by the selector) interchangeably. In the future, these should be referred to as the *selector* and the *selector target*, respectively.

The main idea involves adding a *selected sessions* list that contains 0 or more sessions and acts like a default set of target overrides for all actions that support them. The *selector target* will not be automatically added/removed from the *selected sessions* list when the *selector* is moved; instead, it would be added/removed via hotkeys. When you don't have multiple sessions selected, hotkey actions that support target overrides (and don't have any configured) affect the *selector target* directly.  
The Next/Previous Session hotkey actions would be renamed and continue to move the *selector* as they did previously, the only difference being that whatever the *selector* is pointing at would not be selected automatically when other sessions are already selected.  

Implementation would involve adding a new setting called something like "TargetSessions" that is a list of `TargetInfo` structs. The current "TargetSession" setting would instead refer to the *selector target*. When a session is selected, its `TargetInfo` is added to the "TargetSessions" list, and when it is deselected, its `TargetInfo` would be removed.

Some behaviour examples:
```
1. *selected sessions* list is empty
2. *selector* is pointing at a session
3. hotkey action that supports target overrides is triggered
4. action affects the *selector target*

1. *selected sessions* list is empty
2. *selector* is **not** pointing at a session
3. hotkey action that supports target overrides is triggered
4. action doesn't do anything

1. *selected sessions* list is not empty
2. *selector* is pointing at a session not included in *selected sessions*
3. hotkey action that supports target overrides is triggered
4. action affects the sessions in *selected sessions*, but not the *selector target*
```

**Comment by radj307** [comment]
Implemented a working multi-selector class in https://github.com/radj307/volume-control_audioAPI-test/commit/f0124c986948442b662cb53097fd2b196e7aa09a.  
The next step would be to port this to Volume Control and wire up the viewmodels, hotkeys, and GUI to take advantage of this implementation.

**Comment by radj307** [comment]
Implemented in 88ed96dd3e46c8251f879e7595d076b6d90ee7ac

**Comment by radj307** [comment]
This feature is now available for testing in [v6.1.0 Preview 1](https://github.com/radj307/volume-control/releases/tag/6.1.0-pr1)

**Comment by radj307** [comment]
This feature is now available in [v6.1.0](https://github.com/radj307/volume-control/releases/tag/6.1.0).
